src/libtrx/include/libtrx/game/lara/types.h

```diff
@@ -84,10 +84,9 @@ typedef struct {
         XYZ_32 initial_pos;
     } interact_target;
 
-#if TR_VERSION == 1
     LARA_GUN_TYPE holsters_gun_type;
     LARA_GUN_TYPE back_gun_type;
-#else
+#if TR_VERSION >= 2
     LARA_GUN_TYPE last_gun_type;
     GAME_OBJECT_ID back_gun_obj_id;
```

Comment:
Can we remove `back_gun_obj_id` or will that be a part of a future PR?

Reply:
I think it should be OK, but yes I think I'll tackle it separately for safety given how fragile these meshes are 😄